README.md

```diff
@@ -27,15 +27,35 @@ BAR Lobby is a new WIP lobby for the RTS game [Beyond All Reason](https://www.be
 
 ### Recommended Environment
 
-It is highly recommended to use [VSCode](https://code.visualstudio.com/) for development, as it provides full, built-in TypeScript support, as well as useful extensions such as [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and [Volar](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar).
+It is highly recommended to use [VSCode](https://code.visualstudio.com/) for development, as it provides full, built-in TypeScript support, as well as useful extensions such as [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and [Vue - Official (previously Volar)](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar).
 
 ### Requirements
 
--   [Node.js](https://nodejs.org/en/download/)
+-   [Node.js 18](https://nodejs.org/en/download/)
 
-### Project setup:
+#### Installing Node 18 on Bash
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.0/install.sh | bash
+
+nvm install 18
+
+node -v # should print `v18.20.4`
+```
+
+#### Installing Node 18 on Fish
```

Comment:
IMHO, fish is way too obscure to have dedicated section about it in readme, also, another reason why just pointing at nvm docs will be better.

Reply:
in my particular case, the nvm docs only *had* bash instructions, so it took a hot sec to figure it out at all. Totally fair if we don't want to include it though